src/common/util/keycloak.ts

```diff
@@ -0,0 +1,55 @@
+import getConfig from 'next/config'
```

Comment:
@PepiPetrov This file has been moved to `frontend/src/middleware/auth/keycloak.ts` on `master`

There's no point in having two copies of the same file, so please remove this sone and keep only the one from `middleware/auth/keycloak`